### ISSUE TITLE
Wrong Intentation detection for rules with multiple values 

### ISSUE THREAD

**Issue by HectorLS** [sasstools/sass-lint#1263]
Hello,
i would like to have a proper indentation for a property with multiples values, so i want to put each one in it's own line.
But i'm getting the error about wrong indentation, because the values are being detected as properties.

Any idea how to do this, keeping of course the overall indentation for rules and properties.
Thanks in advance

```scss
button {
  background-color: red;
  border: 2px solid black;
  color: black;
  transition:
    background-color 300ms ease-out,
    border-color 300ms ease-out,
    color 300ms linear;

  &:hover {
    background-color: black;
    border-color: red;
    color: red;
  }
}
```

<img width="319" alt="screenshot 2019-03-05 at 14 29 47" src="https://user-images.githubusercontent.com/6190477/53808839-9966d700-3f53-11e9-8467-02fdc2dd50df.png">

_.sass-lint.yml_
```yml
options:
  formatter: stylish
files:
  include: '**/*.s+(a|c)ss'
rules:
  # Extends
  extends-before-mixins: 1
  extends-before-declarations: 1
  placeholder-in-extend: 1

  # Mixins
  mixins-before-declarations:
    - 2
    -
      exclude:
        - breakpoint
        - mq

  # Line Spacing
  one-declaration-per-line: 1
  empty-line-between-blocks: 1
  single-line-per-selector: 0

  # Disallows
  no-attribute-selectors: 0
  no-color-hex: 0
  no-color-keywords: 0
  no-color-literals: 0
  no-combinators: 0
  no-css-comments: 1
  no-debug: 1
  no-disallowed-properties: 0
  no-duplicate-properties: 1
  no-empty-rulesets: 1
  no-extends: 0
  no-ids: 0
  no-important: 1
  no-invalid-hex: 1
  no-mergeable-selectors: 1
  no-misspelled-properties: 1
  no-qualifying-elements:
    - 1
    -
      allow-elements-width-attribute: true
  no-trailing-whitespace: 1
  no-trailing-zero: 1
  no-transition-all: 1
  no-universal-selectors: 0
  no-url-domains: 1
  no-url-protocols: 1
  no-vendor-prefixes:
    - 1
    -
      ignore-non-standard: true
  no-warn: 0
  property-units: 0

  # Nesting
  declarations-before-nesting: 1
  force-attribute-nesting: 1
  force-element-nesting: 1
  force-pseudo-nesting: 1

  # Name Formats
  class-name-format:
    - 1
    -
      convention: hyphenatedbem

  function-name-format: 1
  id-name-format: 0
  mixin-name-format: 1
  placeholder-name-format: 1
  variable-name-format:
    - 1
    -
      convention: hyphenatedbem

  # Style Guide
  attribute-quotes: 1
  bem-depth: 0
  border-zero: 1
  brace-style: 1
  clean-import-paths: 1
  empty-args: 0
  hex-length: 1
  hex-notation:
    - 1
    -
      style: uppercase

  indentation: 1
  leading-zero: 1
  max-line-length: 0
  max-file-line-count: 0
  nesting-depth:
    - 1
    -
      max-depth: 5

  property-sort-order: 1
  pseudo-element: 0
  quotes: 1
  shorthand-values: 1
  url-quotes: 1
  variable-for-property: 1
  zero-unit: 1

  # Inner Spacing
  space-after-comma: 1
  space-before-colon: 0
  space-after-colon: 1
  space-before-brace: 1
  space-before-bang: 1
  space-after-bang: 1
  space-between-parens: 1
  space-around-operator: 1

  # Final Items
  trailing-semicolon: 1
  final-newline: 1
```
